### PR TITLE
DOC 641 stripe update

### PR DESCRIPTION
**Pull request by markzegarelli**
### Proposed changes

Updated Stripe Source doc to include changes cascaded from an API upgrade.

Preview [here](https://deploy-preview-4089--segment-docs.netlify.app/docs/connections/sources/catalog/cloud-apps/stripe)

### Merge timing
Deploy on 1/30/2023

### Related issues (optional)
DOC-641

### CODE REVIEW

File: src/connections/sources/catalog/cloud-apps/stripe/index.md [hunk 1]
```diff
@@ -6,27 +6,25 @@ id: 1bow82lmk
...
+4. Click Connect, and authenticate with Stripe's OAuth.
 
 ## Components
 
 ### Sync
 
-The Stripe source is built with a sync component, which means we'll make requests to their API on your behalf on a 3 hour interval to pull the latest data into Segment. In the initial sync, we'll grab all the Stripe objects (and their corresponding properties) according to the Collections Table below. The objects will be written into a separate schema, corresponding to the source instance's schema name you designated upon creation (ie. `stripe_prod.charges`).
+The Stripe source is built with a sync component, which means Segment make requests to the Stripe API on your behalf on a 3 hour interval to pull the latest data into Segment. In the initial sync, Segment grabs all the Stripe objects (and their corresponding properties) according to the Collections Table below. The objects are written into a separate schema, corresponding to the source instance's schema name you designated upon creation (for example, `stripe_prod.charges`).
```
Comment (pwseg) [inline src/connections/sources/catalog/cloud-apps/stripe/index.md:20]:
```suggestion
The Stripe source is built with a sync component, which means Segment make requests to the Stripe API on your behalf on a three hour interval to pull the latest data into Segment. In the initial sync, Segment grabs all the Stripe objects (and their corresponding properties) according to the Collections Table below. The objects are written into a separate schema, corresponding to the source instance's schema name you designated upon creation (for example, `stripe_prod.charges`).
```

File: src/connections/sources/catalog/cloud-apps/stripe/index.md [hunk 2]
```diff
@@ -6,27 +6,25 @@ id: 1bow82lmk
...
+Segment's sync component uses an upsert API, so the data in your warehouse loaded using sync reflects the latest state of the corresponding resource in Stripe. For example, if `subscription_status` goes from `active` to `inactive` between syncs, on its next sync that ticket's status is `inactive`.
 
-The source syncs and warehouse syncs are independent processes. Source runs pull your data into the Segment Hub, and warehouse runs flush that data to your warehouse. Sources will sync with Segment every 3 hours. Depending on your Warehouses plan, we will push the Source data to your warehouse on the interval associated with your billing plan.
+The source syncs and warehouse syncs are independent processes. Source runs pull your data into the Segment Hub, and warehouse runs flush that data to your warehouse. Sources will sync with Segment every 3 hours. Depending on your Warehouses plan, Segment pushes the Source data to your warehouse on the interval associated with your billing plan.
```
Comment (pwseg) [inline src/connections/sources/catalog/cloud-apps/stripe/index.md:24]:
```suggestion
The source syncs and warehouse syncs are independent processes. Source runs pull your data into the Segment Hub, and warehouse runs flush that data to your warehouse. Sources will sync with Segment every three hours. Depending on your Warehouses plan, Segment pushes the Source data to your warehouse on the interval associated with your billing plan.
```

**Review by pwseg — APPROVED** [review]

File: src/connections/sources/catalog/cloud-apps/stripe/index.md [hunk 3]
```diff
@@ -501,7 +509,21 @@ Segment flattens Stripes `inventory`, `package_dimensions` objects.
 | `plan_id`         | References the associated plan, if available                                    |
 | `quantity`        | The quantity of the plan to which the customer is subscribed                    |
 | `subscription_id` | The ID of the subscription this item belongs to                                 |
+| `tax_rates`       | An array of tax rate IDs.                                                       |
+
 
+### tax_rates
```
Comment (vivek-26) [inline src/connections/sources/catalog/cloud-apps/stripe/index.md:515]: Tax rates collection is not rendered as a table in the [preview](https://deploy-preview-4089--segment-docs.netlify.app/docs/connections/sources/catalog/cloud-apps/stripe).

**Review by vivek-26 — CHANGES_REQUESTED** [review]

File: src/connections/sources/catalog/cloud-apps/stripe/index.md (hunk 3 above)
Comment (markzegarelli) [reply src/connections/sources/catalog/cloud-apps/stripe/index.md:515]: Fixed this and rebuilding the preview now.

**Pull request merged**